starter-code/apple-pie-starter-code/index.html

```diff
@@ -1,35 +1,87 @@
+
 <head>
   <title>Apple Pie Recipe</title>
+  <link rel="stylesheet" type="text/css" href="css/main.css">
+  <link rel="stylesheet" type="text/css" href="css/header.css">
+
 </head>
+
 <body>
-Apple Pies
-This was my grandmother's apple pie recipe. I have never seen another one
-quite like it. It will always be my favorite and has won me several first place
-prizes in local competitions. I hope it becomes one of your favorites as well!
-
-Ingredients
-1 recipe pastry for a 9 inch double crust pie
-1/2 cup unsalted butter
-3 tablespoons all-purpose flour
-1/4 cup water
-3 tablespoons all-purpose flour
-1/4 cup water
-1/2 cup white sugar
-1/2 cup packed brown sugar
-8 Granny Smith apples - peeled, cored and sliced
-
-Directions
-Preheat oven to 425 degrees F (220 degrees C). Melt the butter in a saucepan. Stir in
-flour to form a paste. Add water, white sugar and brown sugar, and bring to a boil. Reduce
-temperature and let simmer.
-
-Place the bottom crust in your pan. Fill with apples, mounded slightly. Cover with a
-lattice work crust. Gently pour the sugar and butter liquid over the crust. Pour slowly so
-that it does not run off.
-
-Bake 15 minutes in the preheated oven. Reduce the temperature to 350 degrees F (175
-degrees C). Continue baking for 35 to 45 minutes, until apples are soft.
+  <header class="header">
+    <div class="header-image flex">
+      <h1>Apple Pies</h1>
+      <!-- <img src="" alt="apple-pie"> -->
+    </div>
+    <article class="intro">
+      <h3>
+        This was my grandmother's apple pie recipe. I have never seen another one
+        quite like it. It will always be my favorite and has won me several first place
+        prizes in local competitions. I hope it becomes one of your favorites as well!
+      </h3>
+    </article>
+  </header>
+
+  <section class="ingredients-section container">
+    <img class="cooking-info" src="images/recipe-info.png" alt="cooking-info">
+    <hr>
+    <h2>Ingredients</h2>
+    <ul class="ingredients">
+      <li class="ingredient">
+        – 1 recipe pastry for a 9 inch double crust pie
+      </li>
+      <li class="ingredient">
+          – 1/2 cup unsalted butter
+      </li>
+      <li class="ingredient">
+          – 3 tablespoons all-purpose flour
+      </li>
+      <li class="ingredient">
+          – 1/4 cup water
+      </li>
+      <li class="ingredient">
+          – 1/4 cup water
+      </li>
+      <li class="ingredient">
+          – 3 tablespoons all-purpose flour
+      </li>
+      <li class="ingredient">
+          – 1/2 cup white sugar
+      </li>
+      <li class="ingredient">
+          – 1/2 cup packed brown sugar
+      </li>
+      <li class="ingredient">
+          – 8 Granny Smith apples - peeled, cored and sliced
+      </li>
+    </ul>
+  </section>
+
+  <section class="directions-section container">
+      <img class="cooking-info" src="images/cooking-info.png" alt="cooking-info">
+    <hr>
+
+    <h2>Directions</h2>
+    <ul class="directions">
```

Comment:
Here, you should have used an `<ol></ol>` tag instead of ul. Your directions are always going be ordered